.gitignore

```diff
@@ -5,4 +5,7 @@ minimp3
 minimp3_arm
 *.gcda
 *.gcno
-*.gcov
\ No newline at end of file
+*.gcov
+
+fuzzing/afl*
```

Comment:
Is there a reason these need to be ignored?

Reply:
Ah no, that was for my personal usage; I'll delete this.